roles/metrics/README.md

```diff
@@ -0,0 +1,25 @@
+theforeman.operations.metrics
+=============================
+
+Setup Performace Co-Pilot and Grafana to grather metrics of a Foreman installation.
+
+This role is heavily based on the [performancecopilot.metrics](https://github.com/performancecopilot/ansible-pcp) collection.
+
+Role Variables
+--------------
+
+* `foreman_metrics_url`: The URL of the OpenMetrics endpoint of Foreman, defaults to `https://{{ ansible_fqdn | default('localhost') }}/metrics`.
```

Comment:
is the endpoint `/metrics` actually changeable?

Reply:
In Foreman? no.

But you could use this role to monitor a remote Foreman.

Reply:
Where I was going with it was, should we only allow users to configure the hostname rather than the full URL if the `/metrics` part is not configurable?

Reply:
Technically they also have the choice of http vs https (if they disable https enforcement)